kuka_ros2_control_sim_support/launch/test_common_gazebo_classic_ros2_control.launch.py

```diff
@@ -0,0 +1,298 @@
+                "kuka_kr3_support", 
+                "kuka_kr5_support", 
+                "kuka_kr6_support", 
+                "kuka_kr10_support", 
+                "kuka_kr16_support",
+                "kuka_kr120_support",
+                "kuka_kr150_support",
+                "kuka_kr210_support",
+                ],
+            description="Description package with robot URDF/xacro files. Usually the argument \
+        is not set, it enables use of a custom description.",
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "robot_description_macro_file",
+            choices=[
+                "kr3r540_macro.xacro", 
+                "kr5_arc_macro.xacro", 
+                "kr6r700sixx_macro.xacro", 
+                "kr6r900_2_macro.xacro", 
+                "kr6r900sixx_macro.xacro",
+                "kr10r900_2_macro.xacro",
+                "kr10r1100sixx_macro.xacro",
+                "kr10r1420_macro.xacro",                
+                "kr16_2_macro.xacro", 
+                "kr120r2500pro_macro.xacro", 
+                "kr150_2_macro.xacro", 
+                "kr150r3100_2_macro.xacro", 
+                "kr210l150_macro.xacro",
+                ],
+            description="URDF/XACRO description file with the robot.",
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "prefix",
+            default_value="",
+            description="Prefix of the joint names, useful for \
+        multi-robot setup. If changed than also joint names in the controllers' configuration \
+        have to be updated.",
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "robot_controller",
+            default_value="forward_position_controller",
+            choices=["forward_position_controller", "joint_trajectory_controller"],
+            description="Robot controller to start.",
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "use_mock_hardware",
+            default_value="true",
+            description="Start robot with fake hardware mirroring command to its states.",
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "ros2_control_sim_support_package",
+            default_value="kuka_ros2_control_sim_support",
+            description="Support package for ros2_control and simulators. Contains files that use base "
+            "kuka robot description and add on ros2_control support with gazebo simulators. ",
+        )
+    )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "common_sim_description_file",
+            default_value="common_sim_kuka.xacro",
+            description="loads robot macros by chosen value for arguments 'robot_description_package'"
+            "'robot_description_file' and 'robot_name'. Also loads ros2_control with simulators support .",
+        )
+    )
```

Comment:
I don't think this is needed, really. If someone wants to change this, they will probably have their launch file anyway.